frontend/src/scenes/cohorts/Cohorts.tsx

```diff
@@ -59,7 +59,11 @@ const cohortsUrlLogic = kea<cohortsUrlLogicType>({
         },
     },
     actionToUrl: ({ values }) => ({
-        setOpenCohort: () => '/cohorts' + (values.openCohort ? '/' + (values.openCohort.id || 'new') : ''),
+        setOpenCohort: () =>
+            combineUrl(
+                '/cohorts' + (values.openCohort ? `/${values.openCohort.id || 'new'}` : ''),
```

Comment:
Suggestion/nit: let's casually move any encountered urls to `urls.something()`

Reply:
Good point!